template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitattributes/.gitattrbutes

```diff
@@ -0,0 +1,107 @@
+## Set Git attributes for paths including line ending
+## normalization, diff behavior, etc.
+##
+## Get latest from `dotnet new gitattributes`
+
+# Auto detect text files and perform LF normalization
+* text=auto
+
+#
+# The above will handle all files NOT found below
+#
+
+*.cs     text diff=csharp
+*.cshtml text diff=html
+*.csx    text diff=csharp
+*.sln    text eol=crlf
+
```

Comment:
```suggestion
*.fs    text
*.fsi   text
*.fsx   text

```

Reply:
I'm struggling to define a principle for what to include and what to leave out here.

I _believe_ that `*.fs` and `*.vb` and such will already be treated as text, so I don't think with the current template those provide any additional value. The C# ones are only to specify the diff pattern.

I could be convinced to add them if either:

1. There's some benefit we can describe to explicitly marking them as text and not relying on Git to sniff, in which case we need to decide if we want to include stuff that's not .NET proper but often used by .NET (e.g. `.props` and `.targets`)
2. They get miscategorized by Git, in which case the rationale is clear

Let me know if this makes sense, or if there's something I'm missing. Thanks!

Reply:
Ah, this logic makes perfect sense. Good thinking.